src/gwc/pom.xml

```diff
@@ -85,6 +85,10 @@
       <groupId>org.geowebcache</groupId>
       <artifactId>gwc-diskquota-jdbc</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.geowebcache</groupId>
+      <artifactId>gwc-aws-s3</artifactId>
```

Comment:
Should this maybe be a plugin instead of in the core GS GWC module?  How heavy are the AWS dependencies that get pulled in as a result?